package.json

```
@@ -15,7 +15,7 @@
     "publish-docs": "GIT_USER=oblador yarn workspace @loki/website publish-gh-pages"
   },
   "engines": {
-    "node": ">=14.0.0"
+    "node": ">=16.0.0"
```

```suggestion
    "node": ">=16"
```

---

I used https://semver.npmjs.com/ to compare the matches
_Package: `node`, Version range: `>=16`_

---

Shouldn't we have at least the same version that we have in the `.nvmrc`?

```suggestion
    "node": ">=16.20.0"
```

---

@techeverri the `.nvmrc` is for development, but the `engines` is for consumers. There's no strict reason they need to be identical.